firmware/qemu/src/bin/log.rs

```diff
@@ -308,6 +308,9 @@ fn main() -> ! {
 
     defmt::info!("after nested log: {:?}", nested());
 
+    // printing @ is now allowed
+    defmt::info!("I can now print the @ symbol!");
```

Comment:
Could you also add a test for `intern!` macro? Something like lines 66 and 68 but the interned string should contain the  `@` character.